### ISSUE TITLE
release out of beta

### ISSUE THREAD

**Issue by mesqueeb** [alexkuc/chartjs-plugin-doughnutlabel-rebourne#7]
Hey @alexkuc I saw that this library has been in beta for a long time. I also don't see any issues have come up, so it's safe to release it out of beta I think.

Best regards

**Comment by alexkuc** [comment]
@mesqueeb Yeah, you are right. I think it's safe to do. Although… I am thinking of doing a major refactor for this package. In the near future, depending on my free time, I'll either do that or bump the version.
